### PR TITLE
Image Limit Support

### PR DESCRIPTION
**Pull request by Dfmarulanda**
Image Limit Support.
Using Config.Camera.imageLimit you can add a image limit selection for the gallery.

### CODE REVIEW

File: Sources/Utils/Config.swift [hunk 1]
```diff
@@ -11,6 +11,7 @@ public struct Config {
   public struct Camera {
 
     public static var recordLocation: Bool = false
+	public static var imageLimit : Int = 0
```
Comment (JohnSundell) [inline Sources/Utils/Config.swift]: How about making this an optional instead? So that `nil` means "no limit".

File: Sources/Images/Cart.swift [hunk 2]
```diff
@@ -32,12 +32,12 @@ public class Cart {
 
   func add(_ image: Image, newlyTaken: Bool = false) {
     guard !images.contains(image) else { return }
-
-    images.append(image)
-
-    for case let delegate as CartDelegate in delegates.allObjects {
-      delegate.cart(self, didAdd: image, newlyTaken: newlyTaken)
-    }
+	if images.count < Config.Camera.imageLimit  || Config.Camera.imageLimit == 0 {
```
Comment (JohnSundell) [inline Sources/Images/Cart.swift]: Please use the same indentation as the rest of the file uses.

**Comment by Dfmarulanda** [comment]
Hello, i made some improvements, because images from gallery and camera are connected, so, now shutterbutton is disabled when you exceed the limit of images selected. Now imageLimit is a optional value, but, the unwrapping seems weird, but, i think that's the most readable way. I'll implement some test in future, i have no time now. But, i have tested manually.

**Comment by onmyway133** [comment]
@Dfmarulanda Hi, this is already covered by https://github.com/hyperoslo/Gallery/pull/14, which is merged. This will benefit you too. Thanks for your PR

**Pull request closed without merging**